pkg/corerp/frontend/controller/applications/graph_util.go

```diff
@@ -208,11 +210,11 @@ func computeGraph(applicationName string, applicationResources []generated.Gener
 		resources = append(resources, resource)
 
 		// Application-scoped resources are by definition "in" the application
-		resourcesByIDInApplication[*resource.ID] = true
+		resourcesByIDInApplication[to.String(resource.ID)] = true
```

Comment:
This causes potential nil reference panicking. We need to use `to` package to avoid this panic.

Reply:
I wonder if we can use this library in our codebase: https://github.com/uber-go/nilaway.

Reply:
> I wonder if we can use this library in our codebase: https://github.com/uber-go/nilaway.

nilaway is static analysis tool for the code. We can add it to find the potential nil panic as part of workflow. The tool shows the potential problem, not always the real problem, so golangci-lint rejected the proposal to add it to its linter. See this - https://github.com/golangci/golangci-lint/issues/4045